charts/rbac/templates/client/cluster.yaml

```diff
@@ -32,4 +32,18 @@ subjects:
   kind: Group
   name: {{ .Values.rbac.client.name }}:view
 
+---
+kind: ClusterRoleBinding
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: keycloak:developer
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: {{ .Values.rbac.client.name }}-readonly-clusterrole
+subjects:
+- apiGroup: rbac.authorization.k8s.io
+  kind: Group
+  name: developer
```

Comment:
This is pointless binding
you are binding readonly clusterrole to developer, when we have separated `developers` section

I think it's better to stick with same naming: `readonly` oidc group to `readonly` role